server/src/controllers/blogs.js

```diff
@@ -28,12 +34,13 @@ exports.findBlog = async(req, res) => {
     // Update a blog
 exports.updatedBlog = async(req, res) => {
         try {
+            result = await blogValidationSchema.validateAsync(req.body)
             const blog = await Blog.findById(req.params.id)
             Object.assign(blog, req.body)
             await blog.save()
             res.send({ data: blog })
-        } catch {
-            res.status(404).send({ error: "Blog not found!" })
+        } catch (error) {
```

Comment:
404 is Not found error and u are using it everywhere.
Use status code related to the error
ex: 404 Not found, 400 Bad Request, 500 Internal Server Error

Reply:
It's okay. Let me correct.